### DISCUSSION TITLE
Apple decommissioning XML feed

### DISCUSSION THREAD

**Discussion by gosodo** [fastlane/fastlane#20868] [Q&A] [answered]
We got an email about apple decommissioning XML feed and the switch to use AppStore connect API. This seems to affect Transporter which uses XML feed to upload binaries and Metadata. I noticed you use Transporter to make uploads to AppStore. Is this expected to have an impact on Fastlane? Is there a plan to move to the API incase this will impact Fastlane? Also does passing the --api_key_path parameter change the underlying mechanism to using app store connect Api by any chance?

**Comment by mdgeist** [comment, marked as answer]
In version `2.210.0` of Fastlane they switched from using ITSM Transporter to `altool` for uploading to Testflight. This was done because of an issue with ITSM Transporter and Xcode 14.X versions. 
Heres the PR with more info: https://github.com/fastlane/fastlane/pull/20631

Also you can't upload binaries with the App Store Connect API, you can use the JWT used with the API to upload with the `altool`. If thats what you meant. This doc from Apple spells out what you can upload with:
see for more info https://help.apple.com/app-store-connect/#/devb1c185036

**Reply by gosodo** [reply]
Thank you for your prompt response.
